routes/vote/vote.controller.js

```diff
@@ -29,19 +29,19 @@ const postOptIn = (req, res, next) => {
   req.session.vote.confirmOptIn = confirmOptIn
 
   // if yes, go to second page of vote
-  // if no, go to confirmation
-  if (confirmOptIn == 'No') {
-    req.session.vote.voterCitizen = null
-    req.session.vote.voterConsent = null
-    req.session.vote.voterPageEdited = 0
-    return res.redirect('/checkAnswers')
+  if (confirmOptIn == 'Yes') {
```

Comment:
Let's use strict equals (`===`) all the time so that we don't end up with weird casting problems down the road.